lib/api.go

```diff
@@ -107,7 +107,7 @@ const (
 	// fsi endpoints
 
 	// AEStatus returns the filesystem dataset status
-	AEStatus = APIEndpoint("/status/{path:.*}")
+	AEStatus = APIEndpoint("/status")
```

Comment:
It can, but the route expansion is handled by `handleRefRouteMethods` in `api/api.go` where it registers all the variants, this basically just represents the "route prefix" as we used to carry around.